.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,15 @@
+name: Pre-commit
+on: [push]
+jobs:
+  gdformat:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.x'
+      - name: Install pre-commit
+        run: pip install pre-commit
```

Comment:
We could probably pin to a version, but meh.

Reply:
So you're telling me it's just NixOS that needs to install dependencies (gdtoolkit) for relevant pre-commit hooks to work? :face_exhaling: